examples/layertree.js

```diff
@@ -25,11 +25,6 @@ var app = {};
 app.module = angular.module('app', ['ngeo']);
 
 
-// Use the default "popup" template.
-app.module.value('ngeoPopupTemplateUrl',
```

Comment:
Is the https://github.com/camptocamp/ngeo/blob/master/examples/partials/layertree.html not used anymore then ?